### PR TITLE
HubSpot Cloud Extract Documentation Updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated HubSpot documentation.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [HubSpot documentation update](https://www.notion.so/rudderstacks/HubSpot-documentation-update-2eb69d28bf194174982775dd3f9d05f7)

### CODE REVIEW

File: docs/cloud-extract-sources/hubspot/index.mdx [hunk 1]
```diff
@@ -1,106 +1,81 @@
...
-<!--<img src="../assets/1 (15) (1) (1).png" />-->
+## Getting started
 
-<img src="../../assets/1 (15) (1) (1).png" alt="Rudderstack dashboard" />
+To set up HubSpot as a source in RudderStack, follow these steps:
 
-- Then, click on the **Directory** option on the left navigation bar and go to **Cloud Extract** under **Sources**.
+1. Log into your [RudderStack dashboard](https://app.rudderstack.com/).
+2. Go to **Sources** > **New source** > **Cloud Extract** and select **HubSpot** from the list of sources.
+3. Assign a name to your source and click on **Next**.
```
Comment (ishachopra0401) [inline docs/cloud-extract-sources/hubspot/index.mdx:24]:
```suggestion
3. Assign a name to your source and click on **Continue**.
```
Just struck me while going through the web app.. may be we can use 'Continue' going forward..

File: docs/cloud-extract-sources/hubspot/index.mdx [hunk 2]
```diff
@@ -1,106 +1,81 @@
...
-- From the list of sources, click on **HubSpot**.
-- Assign a name to your source, and click on **Next**.
+<div class="infoBlock">
 
-<!--<img src="../assets/3 (13).png" />-->
+Your HubSpot account and the related details will then automatically appear under <strong>Choose an account</strong>.
+</div>
 
-<img src="../../assets/3 (13).png" alt="Hubspot" />
+2. Select the appropriate **HubSpot software** from the dropdown.
```
Comment (ishachopra0401) [inline docs/cloud-extract-sources/hubspot/index.mdx:35]: This setting appears above the 'Connect with Hubspot' option. May be we can move this as step 1

**Review by ishachopra0401 — APPROVED** [review]

File: docs/cloud-extract-sources/hubspot/index.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/cloud-extract-sources/hubspot/index.mdx:35]: Makes sense - done!

File: docs/cloud-extract-sources/hubspot/index.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/cloud-extract-sources/hubspot/index.mdx:24]: Yep - good catch! done.

File: docs/cloud-extract-sources/hubspot-v2.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
+---
+#slug: "/docs/cloud-extract-sources/hubspot"
+title: "HubSpot"
+description: Step-by-step guide on ingesting data from HubSpot into RudderStack.
+---
+
+# HubSpot Source
```
Comment (ameypv-rudder) [inline docs/cloud-extract-sources/hubspot-v2.mdx:7]:
```suggestion
# HubSpot v2
```

File: docs/cloud-extract-sources/hubspot-v2.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+[HubSpot](https://www.HubSpot.com/) is a leading marketing and CRM platform that helps you track leads as well as inbound marketing and sales.
+
+This document guides you in setting up HubSpot as a source in RudderStack. Once configured, RudderStack automatically ingests your specified HubSpot data and routes it to your data warehouse.
+
+<div class="warningBlock">
+
+All the Cloud Extract sources support sending data only to a <a href="https://www.rudderstack.com/docs/data-warehouse-integrations/">data warehouse destination</a>.
+</div>
+
```
Comment (ameypv-rudder) [inline docs/cloud-extract-sources/hubspot-v2.mdx:17]:
```suggestion
<img src="https://img.shields.io/badge/Stability-Beta-blueviolet" alt="Github Badge" class="githubBadge" />

```

**Review by chaitanyapraveen — APPROVED** [review]
LGTM!

File: docs/cloud-extract-sources/hubspot-v2.mdx [hunk 5]
```diff
@@ -0,0 +1,88 @@
+---
+#slug: "/docs/cloud-extract-sources/hubspot"
+title: "HubSpot"
+description: Step-by-step guide on ingesting data from HubSpot into RudderStack.
+---
+
+# HubSpot v2
```
Comment (ishachopra0401) [inline docs/cloud-extract-sources/hubspot-v2.mdx:7]:
```suggestion
# HubSpot v2 Source
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**